src/codesyncd/utils.ts

```diff
@@ -197,11 +198,19 @@ export class statusBarMsgs {
 		if (!fs.existsSync(this.settings.CONFIG_PATH)) return STATUS_BAR_MSGS.NO_CONFIG;
 		const repoPath = pathUtils.getRootPath();
 		const daemonError = CodeSyncState.get(CODESYNC_STATES.DAEMON_ERROR);
-		const userState = new UserState().get();
+		const userState = new UserState();
+		const user = new UserState().get();
+		// Login is in process
+		if (user.isWaitingForLogin) {
+			const authInProcess = CodeSyncState.canSkipRun(CODESYNC_STATES.USER.AUTHENTICATION_INITIATED_AT, AUTHENTICATION_TIMEOUT);
```

Comment:
Any idea why this statement is not covered? any way we can cover it?

Reply:
I think we'll have to add unit tests for this function, I think we have few cases but probably not covering each line in this function (different cases)